src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx

```diff
@@ -74,11 +75,32 @@ export default class Summary extends React.Component {
     });
   }
 
-  showApplicationSummary() {
+  showExitDiagnostics() {
     const {jobInfo} = this.props;
-    this.showEditor('Application Summary', {
+    const messages = jobInfo.jobStatus.appExitMessages;
+    const result = [];
+    if (!isEmpty(messages.container)) {
+      result.push('[Container Stderr]');
+      result.push('');
+      result.push(messages.container);
+      result.push('');
+    }
+    if (!isEmpty(messages.runtime)) {
+      result.push('[Runtime Output]');
+      result.push('');
+      result.push(yaml.safeDump(messages.runtime));
+      result.push('');
+    }
+    if (!isEmpty(messages.launcher)) {
+      result.push('[Launcher Output]');
+      result.push('');
+      result.push(messages.launcher);
+      result.push('');
+    }
+
```

Comment:
The display importance should be:

appExitCode > appExitSpec > (runtime msg > container msg > launcher msg) > trigger

Reply:
And can we show them in different text box?

---
In reply to: [277196097](https://github.com/Microsoft/pai/pull/2550#discussion_r277196097) [](ancestors = 277196097)